File: java/estrutura-sequencial/Consumo.java

import entities.ObterConsumo;
import java.util.Locale;
import java.util.Scanner;

public class Consumo {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        ObterConsumo consumo = new ObterConsumo();

        System.out.print("Distancia percorrida: ");
        consumo.distancia = sc.nextInt();
        System.out.print("Combustível gasto: ");
        consumo.combustivel = sc.nextDouble();

        System.out.println(consumo);

        sc.close();
    }
}

File: js/estrutura-sequencial/consumo.js
const readline = require('readline-sync');

const distancia = Number(readline.question("Distancia percorrida: "));
const combustivel = Number(readline.question("Combustivel gasto: "));
const consumo = distancia / combustivel;

console.log(`Consumo medio = ${consumo.toFixed(3)}`);